src/main/java/technical/SaveLine.java

```diff
@@ -0,0 +1,182 @@
+package technical;
```

Comment:
good use of OOP, by having the job of saving the input details into a line handled by a separate class :)